VideoLocker/res/values/dimens.xml

```diff
@@ -59,7 +59,13 @@
     <dimen name="signup_social_title_margin_large">8dp</dimen>
     <dimen name="signup_social_title_margin_bottom">1dp</dimen>
 
+    <dimen name="edx_hairline">1px</dimen>
```

Comment:
In that case it seems too thin and variable a size to use as for the whole border, as it seems to be used here. Maybe we should use fractional DIPs instead? Or use another dimen resource with a fixed size for defining the border?